service/gcs/runtime/runc/runc.go

```diff
@@ -587,7 +588,15 @@ func (c *container) startProcess(tempProcessDir string, hasTerminal bool, stdioS
 	}
 
 	if err := cmd.Run(); err != nil {
-		return nil, errors.Wrapf(err, "failed to run runc create/exec call for container %s", c.id)
+		b, readErr := ioutil.ReadFile(logPath)
+		if readErr != nil {
+			return nil, errors.Wrapf(readErr, "failed to read runc log for runc create/exec call failure for container %s", c.id)
+		}
+		var result map[string]interface{}
```

Comment:
Isnt this just a logrus entry array?

Reply:
I mean, can it be more than one?

Reply:
Oh I see. There could be more than just an error in here. I wonder if we want to do some wrapping up to the last error for the extended info?